README.md

```diff
@@ -1 +1,28 @@
-## Harmonise music pieces given a soprano or bass voice
+harmonise.it
+============
+
+> Practice music theory harmony rules
+
+harmonise.it is a platform which helps users - music theory harmony students -
+practice on harmony rules. Based on a theme with a given soprano or bass voice,
```

Comment:
I think a syntax like this flows better: "which helps music theory and harmony students practice harmony rules"